visual-espresso/visual/src/main/java/com/saucelabs/visual/exception/VisualApiException.java

```diff
@@ -0,0 +1,12 @@
+package com.saucelabs.visual.exception;
+
+public class VisualApiException extends RuntimeException {
```

Comment:
Runtime Exception is a special exception - are you sure you want to extend the Runtime exception?  Throwing such an exception should kill the test suite. I think we want the test to fail instead?
```suggestion
public class VisualApiException extends Exception {
```

Reply:
This doesn't kill the suite but fails the test instead. Same approach we took in java-client. Tested and works fine.